test/plugin/test_out_google_cloud.rb

```diff
@@ -346,6 +346,13 @@ def test_statusz_endpoint
     end
   end
 
+  def test_monitoring_config
+    # Verify that driver creation succeeds when monitoring type is not
+    # "prometheus" (in which case, we simply don't record metrics).
```

Comment:
You could assert that no prometheus counters have been defined (i.e., that they're `nil`). See `assert_prometheus_metric_value` in `base_test.rb` for implementation ideas.

Reply:
Done.